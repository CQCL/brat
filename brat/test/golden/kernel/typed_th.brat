id :: { Qubit -o Qubit }
id = x => x

id2 :: { Qubit, Qubit -o Qubit, Qubit }
id2 = { let f = {id;id} in q,q' => f(q),f(q') }
