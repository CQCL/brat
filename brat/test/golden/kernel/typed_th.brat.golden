Error in test/golden/kernel/typed_th.brat@5:17
id2 = { let f = {id;id} in q,q' => f(q),f(q') }
                ^^^^^^^

  Thunks not allowed in kernels: {:id()(「id()」):}

